### ISSUE TITLE
[data2vec] On the regularization and normalization of targets 

### ISSUE THREAD

**Issue by tomresan** [facebookresearch/fairseq#4169]
First of all, great work with data2vec!
I am not sure, if this is the correct place to ask such questions. I'd like to experiment with some target regularization techniques in the audio domain and would be happy about some insights and discussions from the authors (@alexeib). I have a few questions regarding this topic:

**1.** Why would layer norm work in the NLP and visual domain, but not in the audio domain? My intuition is that (as mentioned in the paper) audio timesteps are highly correlated. Therefore we need a method to "drag them apart" in the time dimension, which is what instance norm is doing. Is this intuition correct?
**2.** Have you tried to use batch normalization? Why did you stick with instance normalization?
**3.** Have you tried more complex whitening techniques (propsed in e.g. the VIbCReg or W-MSE papers)?
**4.** The paper mentions that you have tried variance and covariance losses from the VICReg paper. How did you implement these ideas? Did you apply the losses on the predictions or on the targets (in the latter case, I'd imagine that you cannot detach the targets as it is currently implemented)? Why are such regularization techniques less reliable with larger models? Is the teacher EMA still necessary when those loss functions are applied?
